ts/sagas/startup/watchMessageLoadSaga.ts

```diff
@@ -0,0 +1,63 @@
+import { TypeofApiCall } from "italia-ts-commons/lib/requests";
+import { buffers, Channel, channel } from "redux-saga";
+import { call, fork, take } from "redux-saga/effects";
+import { ActionType, getType } from "typesafe-actions";
+
+import { GetUserMessageT } from "../../../definitions/backend/requestTypes";
+import { loadMessage as loadMessageAction } from "../../store/actions/messages";
+import { loadMessage } from "../messages/messages";
+
+// Here we set the number of handlers we want to create.
+// This means that we will have at most a number of concurrent
+// fetchs (of the message detail) equal to the number of the handlers.
+const NUMBER_OF_HANDLERS = 5;
```

Comment:
e better term for handlers that process queued items would be _workers_:

```suggestion
const TOT_FETCH_WORKERS = 5;
```

Reply:
also I would move this conf to the env along with other configs